### PR TITLE
[T2A2-A5][T15-A01] Murray Chen

### PR DESCRIPTION
**Pull request by mchen14**
All the commits combined into one pr, am not sure how to separate them. 
Ready for Review though.

### CODE REVIEW

File: src/seedu/addressbook/AddressBook.java [hunk 1]
```diff
@@ -400,6 +400,10 @@ private static String executeAddPerson(String commandArgs) {
         return getMessageForSuccessfulAddPerson(personToAdd);
     }
 
+	private static boolean isValidPerson(final Optional<String[]> decodeResult) {
```
Comment (okkhoy) [inline src/seedu/addressbook/AddressBook.java:433]: once you create a method, you need to add the header comments as well.

File: src/seedu/addressbook/AddressBook.java [hunk 2]
```diff
@@ -577,12 +577,17 @@ private static void executeExitProgramRequest() {
     private static String getUserInput() {
         System.out.print(LINE_PREFIX + "Enter command: ");
         String inputLine = SCANNER.nextLine();
-        // silently consume all blank and comment lines
+        inputLine = consumeBlankAndCommentLines(inputLine);
+        return inputLine;
+    }
+
+	private static String consumeBlankAndCommentLines(String inputLine) {
```
Comment (okkhoy) [inline src/seedu/addressbook/AddressBook.java:619]: need to add header comments as well on creating a new method.

**Comment by okkhoy** [comment]
@mchen14 please rename your pull request according to the right team name,
- for the next tutorial onwards, make separate pull requests for each activity.

**Comment by mchen14** [comment]
just to clarify, how does the tutorial label go? I put T04-A1 because I'm on team A1 in Tutorial group 4; is that not correct?

**Comment by okkhoy** [comment]
No. That is not correct.

If you attended the tutorial you would know that we assigned tutorial and team ID based on the tutorial slot. E.g. Wednesday 9 am tutorial is identified using W09.

Even if you haven't you should take a look a the module website which has clear instructions on how the tutorials are referenced.

**Comment by okkhoy** [comment]
:+1:

**Pull request closed without merging**